tide/master/MasterApplication.cpp

```diff
@@ -406,6 +411,13 @@ void MasterApplication::_initTouchListener()
     connect(_touchListener.get(), &MultitouchListener::touchPointAdded,
             [this, getWallPos](const int id, const QPointF normalizedPos) {
                 _markers->addMarker(id, getWallPos(normalizedPos));
+#if TIDE_ENABLE_PLANAR_CONTROLLER
+                if (_planarController->getState() == screenState::OFF)
+                    if (!_planarController->powerOn())
+                        put_flog(LOG_INFO,
+                                 "Could not power on the screens"
```

Comment:
"Could not power on the screens by " << and don't forget to add a space before "